server/index.js

```diff
@@ -90,16 +100,26 @@ const addSession = async (app) => {
     }
   });
 };
-
+const addLocalization = (app) => {
+  i18next
+    .init({
+      lng: 'ru',
+      fallbackLng: 'en',
+      debug: isDevelopment,
+      resources: { ru },
+    });
+  app.decorate('t', (key) => i18next.t(key));
```

Comment:
when we use i18next.t - everybody would understand what it does. But when we use app.t it's not clear at all